Documentation/Index.rst

```diff
@@ -21,26 +21,20 @@ __________________________________________________
 Getting Started
 ===============
 
-To find out more information about the different versions of TYPO3 and its system requirements, visit https://get.typo3.org.
+* Follow the :ref:`quick installation guide <t3install:install-via-composer>`
+to install TYPO3 using Composer.
 
-Follow the :ref:`Quick installation guide with Composer <t3install:install-via-composer>`
-to install TYPO3.
+* The :ref:`t3start:start` introduces you to TYPO3's backend - the interface used for managing
+content and configuring your TYPO3 installation.
 
-.. code-block:: bash
+* Add a domain, set up languages and configure URL handling with the
+:ref:`site configuration <sitehandling-basics>` tutorial.
 
-   composer create-project typo3/cms-base-distribution YourNewProjectFolder
```

Comment:
We already talked about removing the command via Slack: I agree it is not very useful without much context. But I think I added it in the first place due to a suggestion by someone else. 

This is what you often see in other documentation projects: You already have basic steps on the start page.  So I think this is not such a bad idea. 

I agree with the other changes.